webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -49,7 +49,7 @@
             <li>
                 <label for="products_select">Product:</label>
                 <select id="products_select">
-                {% for p in releases %}
+                {% for p in active_versions %}
```

Comment:
This, and line 64, are thee most important uses of `ProductVersions`. It drives the nav bar on every page.